book/src/3.basics/2.on-message.md

```diff
@@ -1,12 +1,13 @@
 # 👂 Listening To Message Activities
 
-You can listen/subscribe to messages using static text or `Regexp`.
+You can listen/subscribe to messages using static text or `Regexp`. This is a shorthand for `app.on('message')` activity handler, where it will only trigger the handler if it matches the pattern you have provided. These types of handlers are common enough that we have provided a shorthand for them.
```

Comment:
Message is a kind of Activity.  (This is the opposite of what I expect coming from general message based systems)